src/disco/components/Addon.js

```diff
@@ -281,7 +253,9 @@ export class AddonBase extends React.Component {
                 sanitizeHTMLWithExternalLinks(heading, ['a', 'span'])
               }
             />
-            {this.getDescription()}
+            {type !== ADDON_TYPE_THEME &&
```

Comment:
It might be better to move this condition into the `getDescription` function, so that it could just return nothing if the add-on is a theme.